test/UserAuthorization.js

```diff
@@ -21,87 +21,57 @@ let accessTokenAdminIngestor = null,
-        userIdIngestor = idVal;
-        utils.getIdAndToken(
-          appUrl,
-          {
-            username: "user1",
-            password: TestData.Accounts["user1"]["password"],
-          },
-          (idVal, tokenVal) => {
-            accessTokenUser1 = tokenVal;
-            userIdUser1 = idVal;
-            utils.getIdAndToken(
-              appUrl,
-              {
-                username: "user2",
-                password: TestData.Accounts["user2"]["password"],
-              },
-              (idVal, tokenVal) => {
-                accessTokenUser2 = tokenVal;
-                userIdUser2 = idVal;
-                utils.getIdAndToken(
-                  appUrl,
-                  {
-                    username: "user3",
-                    password: TestData.Accounts["user3"]["password"],
-                  },
-                  (idVal, tokenVal) => {
-                    accessTokenUser3 = tokenVal;
-                    userIdUser3 = idVal;
-                    utils.getIdAndToken(
-                      appUrl,
-                      {
-                        username: "user4",
-                        password: TestData.Accounts["user4"]["password"],
-                      },
-                      (idVal, tokenVal) => {
-                        accessTokenUser4 = tokenVal;
-                        userIdUser4 = idVal;
-                        utils.getIdAndToken(
-                          appUrl,
-                          {
-                            username: "archiveManager",
-                            password: TestData.Accounts["archiveManager"]["password"],
-                          },
-                          (idVal, tokenVal) => {
-                            accessTokenArchiveManager = tokenVal;
-                            userIdArchiveManager = idVal;
-                            utils.getIdAndToken(
-                              appUrl,
-                              {
-                                username: "admin",
-                                password: TestData.Accounts["admin"]["password"],
-                              },
-                              (idVal, tokenVal) => {
-                                accessTokenAdmin = tokenVal;
-                                userIdAdmin = idVal;
-                                done();
-                              },
-                            );
-                          },
-                        );
-                      },
-                    );
-                  },
-                );
-              },
-            );
-          },
-        );
-      },
-    );
-  });
-
+  beforeEach(async() => {
+    const loginResponseIngestor = await utils.getIdAndToken(appUrl, {
+      username: "adminIngestor",
+      password: TestData.Accounts["adminIngestor"]["password"],
+    });
+    userIdIngestor = loginResponseIngestor.userId
+    accessTokenAdminIngestor = loginResponseIngestor.token
```

Comment:
Extremely minor detail but no `;` at the end of line, not only for these lines, but for all similar changes applied in this file.